### ISSUE TITLE
`test_binary_with_various_inputs` failure

### ISSUE THREAD

**Issue by jpds** [bensadeh/tailspin#177]
`tailspin` currently fails to build on NixOS unstable with the following error:

```shell
tailspin> test test_binary_with_various_inputs ... FAILED
tailspin>
tailspin> failures:
tailspin>
tailspin> ---- test_binary_with_various_inputs stdout ----
tailspin> thread 'test_binary_with_various_inputs' panicked at tests/utils.rs:19:10:
tailspin> Failed to spawn child process: Os { code: 2, kind: NotFound, message: "No such file or directory" }
tailspin> note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
tailspin>
tailspin>
tailspin> failures:
tailspin>     test_binary_with_various_inputs
tailspin>
tailspin> test result: FAILED. 0 passed; 1 failed; 0 ignored; 0 measured; 0 filtered out; finished in 201.65s
tailspin>
tailspin> error: test failed, to rerun pass `--test integration_tests`
```

`nixpkgs` CI job with this can also be found at: https://hydra.nixos.org/build/276702946

**Comment by jpds** [comment]
Nevermind, I took a look at the relevant code and it's a Nix-specific issue.

**Comment by SebastianBalle** [comment]
Did you find a relevant Nix issue? I can't seem to find one tracking this issue.

**Comment by jpds** [comment]
I've already fixed it in #353457.
